ui/apps/platform/src/utils/queryService.js

```diff
@@ -235,14 +230,6 @@ function getFragmentName(entityType, listType) {
             return 'serviceAccountFields';
         case entityTypes.CONTROL:
             return 'controlFields';
-        case entityTypes.CVE:
-            if (entityType === entityTypes.NODE_COMPONENT) {
-                return 'nodeCVEFields';
-            }
-            if (entityType === entityTypes.IMAGE_COMPONENT) {
-                return 'imageCVEFields';
-            }
-            return 'cveFields';
```

Comment:
Commenting here about lines below original 252-253 and changed 239-240
Is `case entityTypes.COMPONENT:` obsolete yet?

Similar question for VULN_COMPONENT_LIST_FRAGMENT in `getFragment` function